h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -158,6 +158,30 @@ describe('CreateGroupForm', () => {
     assert.isFalse(wrapper.exists('[data-testid="error-message"]'));
   });
 
+  it("doesn't show a loading state initially", async () => {
+    const { wrapper } = createWrapper();
+
+    await assertInLoadingState(wrapper, false);
+  });
+
+  it('shows a loading state when the create-group API request is in-flight', async () => {
+    const { wrapper } = createWrapper();
+    fakeCallAPI.resolves(new Promise(() => {}));
+
+    wrapper.find('form[data-testid="form"]').simulate('submit');
+
+    await assertInLoadingState(wrapper, true);
+  });
+
+  it('continues to show the loading state after receiving a successful API response', async () => {
```

Comment:
We could change this to avoid the back button issue, since the browser will take care of showing loading indicators during the navigation to the new group page.